itou/templates/search/includes/siaes_search_filters.html

```diff
@@ -14,29 +14,77 @@
             {% if form.departments %}
                 <hr>
                 <fieldset>
-                    {% bootstrap_field form.departments wrapper_class="form-group mb-0" label_class="d-block font-weight-bold mb-3" %}
+                    <div class="form-group mb-0">
+                        <legend class="has-collapse-caret mb-0"
+                                data-bs-toggle="collapse"
+                                href="#collapse_{{ form.departments.name }}"
+                                role="button"
+                                aria-expanded="{% if form.departments.value %}true{% else %}false{% endif %}"
+                                aria-controls="collapse_{{ form.departments.name }}">
+                            {{ form.departments.label | capfirst }}
+                        </legend>
+                        <div class="collapse mt-3 {% if form.departments.value %}show{% endif %}" id="collapse_{{ form.departments.name }}">
```

Comment:
Évite l’espace pour rien :)
```suggestion
                        <div class="collapse mt-3{% if form.departments.value %} show{% endif %}" id="collapse_{{ form.departments.name }}">
```

Reply:
C'est fix J'en étais conscient en le faisant, mais je trouvais ça plus lisible avec l'espace (même si ça laisse  trainer un espace vide dans l'attribut class).